### PR TITLE
Fix bug and change cri selection procedure of cluster-sync

### PR DESCRIPTION
**Pull request by jcanocan**
The command for getting the registry port gets more than one output, which breaks following commands in the deploying scripts. The output is now parsed to pickup just the first port.

Also the container runtime selection was hardcoded, now it uses the cri selection script of kubervirtci.

Further details can be found at [issue 2153](https://github.com/kubevirt/hyperconverged-cluster-operator/issues/2153)
**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
Fix make cluster-sync issue and changes the cri selection procedure
```

### CODE REVIEW

File: cluster/sync.sh [hunk 1]
```diff
@@ -1,15 +1,15 @@
 #!/bin/bash -ex
 
 source ./hack/common.sh
+source ${PWD}/_kubevirtci/cluster-up/cluster/ephemeral-provider-common.sh
```
Comment (0xFelix) [inline cluster/sync.sh:4]: Use same path style as above? `.` instead of `${PWD}`?

File: cluster/sync.sh [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-container_command=podman
-
-registry_port=$(${container_command} ps | grep -Po '(?<=0.0.0.0:)\d+(?=->5000\/tcp)' || echo "")
+container_command=$_cri_bin
```
Comment (0xFelix) [inline cluster/sync.sh:6]: Why not replace `container_command` with `_cri_bin` everywhere?

File: cluster/sync.sh [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
+registry_port=$(${container_command} ps | grep -Po '(?<=0.0.0.0:)\d+(?=->5000\/tcp)' | head -n 1)
```
Comment (0xFelix) [inline cluster/sync.sh:8]: Did you check if the regex works for both podman and docker?

File: cluster/sync.sh (hunk 1 above)
Comment (jcanocan) [reply cluster/sync.sh]: I agree.

File: cluster/sync.sh (hunk 2 above)
Comment (jcanocan) [reply cluster/sync.sh]: Makes sense for me too.

**Review by nunnatsa — APPROVED** [review]

**Comment by nunnatsa** [comment]
Coverage reduction seems not related to this PR, because it does not change code nor unit tests.

/override coverage/coveralls
/approve

**Pull request merged**